### DISCUSSION TITLE
Scaling between tensors

### DISCUSSION THREAD

**Discussion by efan3000** [apache/incubator-mxnet#8439] [Q&A]
Hi,

I added following 2 layers in a classic network:

mask =  mx.symbol.broadcast_div(name='div', *[relu, relu])
relu2 = mx.symbol.broadcast_mul(name='relu2', *[relu, mask])

, and found the loss refuses to decrease. In my knowledge, the networks with or without above layers should be same. I tried similar thing in Caffe, it works. I have no idea why it cannot work in MXNET.

Thanks.

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.

**Comment by vishaalkapoor** [comment]
@efan3000 Would you kindly paste a code sample that reproduces the issue, so that we can best answer your question?
